deepinv/training/testing.py

```diff
@@ -143,18 +145,27 @@ def test(
             else:
                 x_net = model(y, physics_cur)
 
-            if hasattr(physics_cur, "A_adjoint"):
-                if isinstance(physics_cur, torch.nn.DataParallel):
-                    x_init = physics_cur.module.A_adjoint(y)
-                else:
-                    x_init = physics_cur.A_adjoint(y)
-            elif hasattr(physics_cur, "A_dagger"):
-                if isinstance(physics_cur, torch.nn.DataParallel):
-                    x_init = physics_cur.module.A_dagger(y)
+            phys = (
+                physics_cur.module
```

Comment:
I'm going to integrate all this in future PR

Reply:
Should I just leave this as is and then this will be overwritten in #280